pennylane/devices/qubit/simulate.py

```diff
@@ -99,4 +101,6 @@ def simulate(circuit: qml.tape.QuantumScript) -> tuple:
     for op in circuit._ops:
         state = apply_operation(op, state)
 
+    if len(circuit.measurements) == 1:
+        return measure(circuit.measurements[0], state)
```

Comment:
This should probably be updated after the changes in #3769 are merged (see changes to `pennylane/transforms/qfunc_transforms.py`)

Reply:
Something different.  Under the current return types specification, tuples of a single measurement value are squeezed to remove the singleton dimension.